Assets/Scripts/Game/UserInterfaceWindows/DaggerfallInventoryWindow.cs

```diff
@@ -1815,6 +1761,7 @@ private void UpdateItemInfoPanel(DaggerfallUnityItem item)
             itemInfoPanelLabel.SetText(tokens);
         }
 
+        // This never gets called AFAICS, do we even need it with OnPush() working? - Hazelnut
```

Comment:
This event is raised when a new game is started (e.g. load game, new character). The purpose is reset to reset certain global inventory state (such as the currently selected tab) that should not carry over from game to game. This is distinct from OnPush() which is used to set things up whenever window is opened. It is being called, but it's not intended for the use you had here, OnPush() was always the correct place for that. :)

Reply:
Yeah I get what its supposed to do now, although I put my clear info panel in there by mistake. That's how I caught that it's not being called. I put a log statement in there and after loading a new game - nothing is logged to console. Worried I broke something I checked out DFU from before I ever committed anything and the same thing so at least it wasn't me.

Not sure why it's not being called, the event works fine in other places. Just wanted to bring to your attention.